docs/release.md

```diff
@@ -324,9 +324,39 @@ sourced from the delta of PRs merged since the last official release. The text
 from those PRs are sourced directly from the ```release-notes``` section in
 each PRs description.
 
-Below is an example of getting the release notes for v0.31.0-rc.0
+## Using GitHub labels for Release Notes
+
+We use GitHub labels to filter release notes, first using a 'kind/' label and
+second using a 'sig/' label where applicable.
+In some cases, 'kubevirt-bot' applies one or more of these labels automatically,
+however they might need to be manually applied by either the author or a reviewer of the PR.
+
+If a PR matches more than one 'kind/' or 'sig/' label, add all that apply. The release note will be included in whichever takes the greater priority.
+
+As per the PR template, if the PR requires additional action from users switching to the new release, include the string "action required" in the release note. The PR will be flagged accordingly.
+
+## The 'kind/' Labels Used for Release Notes
```

Comment:
How about adding a section explaining how to enable these?

That is:
`/kind enhancement`, `/sig security`, etc

Reply:
From my understanding, someone would need triage rights to be able to apply these labels. Which is fine for a reviewer but it would be better if the author was able to apply it themselves.
@dhiller would it be possible for kubevirt-bot to allow an author to add /kind and /sig labels if they include them in the PR description when they open the PR? (But not in a subsequent comment)